app/views/users/index.html.erb

```diff
@@ -1 +1,3 @@
-<h2>Here is a list of users</h2>
+<% @users.each do |user| %>
+  <%= render 'reuse/user_profile', user: user %>
+<% end %>
```

Comment:
Good job using `app/views/users/_user_profile.html.erb`  instead of  `@objects.each`for a better practice 👍

Reply:
Hello @abdulhamiid 

Is it implying i used the right approach or i should use @objects.each?

Reply:
Hi @skabeo, you have already used a good approach 👍